Util/Time/stepsInDay.m

```diff
@@ -0,0 +1,55 @@
+function [steps] = stepsInDay(tunit)
+% function [steps] = timeQuantisationStep(prec)
+%
+% How many steps are defined in day, for a given time unit.
+%
+%
+% Inputs:
+%
+% tunit - string representing a time/precision unit.
+%
+% Outputs:
+%
+% steps - a floating point number
+%
+% Example:
+%
+% [steps] = timeQuantisationStep('minute')
+% assert(isequal(step,1440);
```

Comment:
step with an s

Reply:
Yeap - function name is wrong too, as well as assertion line.